Gems/Atom/Asset/ImageProcessingAtom/Code/Source/ImageLoader/TIFFLoader.cpp

```diff
@@ -22,7 +22,7 @@ namespace ImageProcessingAtom
 {
     namespace TIFFLoader
     {
-        static constexpr int TiffMaxMessageSize = 1024;
+        [[maybe_unused]] static constexpr int TiffMaxMessageSize = 1024;
```

Comment:
You can probably just move this down inside the `AZ_ENABLE_TRACING` block below instead